modules/issue_tracker/php/issuewatcheranonymizer.class.inc

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * This class implements a data mapper which anonymizes User ID in the Issue
+ * Tracker.
+ *
+ * PHP Version 7
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Tools
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+namespace LORIS\issue_tracker;
+use \LORIS\Data\{Mapper, DataInstance
+};
+
+/**
+ * A IssueWatcherAnonymizer is a \LORIS\Data\Mapper which anonymize User ID
+ * by setting it to true if it belongs to the current user and false if not.
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Tools
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+class IssueWatcherAnonymizer implements Mapper
```

Comment:
This doesn't seem to be doing anonymization, it seems to just be doing a mapping of UserID?

Reply:
Yeah, you're right. Poor choice of word. I'll change it to `IssueWatcherMapper`.